src/optimizer.f90

```diff
@@ -100,6 +100,11 @@ subroutine get_optthr(n,olev,ethr,gthr,maxcycle,acc)
       gthr   = 5.d-5
       maxcycle=n*20
       acc=0.01d0
+   case(p_olev_oniom)
+      ethr   = 1.d-5
+      gthr   = 2.d-3
+      maxcycle=n*2
+      acc=1.00d0
```

Comment:
That's close to the DCOSMO-RS settings called `lax` can't we just use those instead?